frontend/src/js/constants/Constants.js

```diff
@@ -107,15 +108,147 @@ const JOYRIDE_STEPS = [
+            "<span style='color: orange;'>TWO operands</span> (variables or constants) or a ",
+            "<span style='color: green;'>SINGLE variable OPERATOR.</span>"
+        ),
+        selector: '#calc-button-region',
+        position: 'bottom-right',
+        type: 'hover'
+    },
+    {
+        title: 'Variables List',
+        text: "All the available variables that you have loaded are listed here.",
+        selector: '#calc-variable-region',
+        position: 'right',
+        type: 'click'
+    },
+    {
+        title: 'Derived Variable',
+        text: "The final name for the derived variable appears here. VDAT automatically constructs one for you, ".concat(
+            "based on your operations, but you can decide on another name by typing it here."),
+        selector: '#new-variable-name',
+        position: 'left',
+        type: 'click'
+    },
+    {
+        title: 'Calculation Area',
+        text: 'The calculation to be performed will be expressed here.'.concat(
+            '<br />To use a variable, drag it in the green rectangle.'),
+        selector: '#calculation',
+        position: 'right',
+        type: 'click'
+    },
+    {
+        title: 'Regrid Operations',
+        text: "The 'Regrid' drop-down, let's you decide which regridder tool and method you would like to use.".concat(
+            '<br /><span style="color: blue;">Linear</span> is the default regrid operation.'
+        ),
+        selector: '#regrid',
+        position: 'bottom-right',
+        type: 'hover'
+    },
+    {
+        title: 'Compute New Variable',
+        text: "When you are done setting the calculation click the 'Enter' button or simply press 'Enter' ".concat(
+            "to perform the calculation and have it saved as a new derived variable.",
+            "You can use it right away to derive yet another variable."
+        ),
+        selector: '#enter',
+        position: 'bottom-right',
+        type: 'hover'
+    }
+];
+
+const COLORMAP_STEPS = [
+    {
+        title: 'Colormap Editor Tour',
+        text: 'This is a tour of the Colormap Editor and how to use it!',
+        selector: '#colormap-editor-main',
+        position: 'left',
+        type: 'click'
+    },
+    {
+        title: 'Colormap List',
+        text: "This dropdown lets you choose from various colormaps.",
+        selector: '#colormap-dropdown',
+        position: 'right',
+        type: 'click'
+    },
+    {
+        title: 'Create New Colormap',
+        text: "Click this button to create a new copy of the current color-map which you can then modify and save as you like.",
+        selector: '#btn-new-colormap',
+        position: 'right',
+        type: 'click'
+    },
+    {
+        title: 'Color Grid',
+        text: 'All the colors of the color map are listed here in order. Clicking on a cell will allow you to change its color.'.concat(
```

Comment:
The colors and their respctive index are listed here [...] will allow you to adjust the color (see next frame) [...]